.github/tests/production-example/install.sh

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env bash
 
-set -x
+set -xe
 
 SCRIPT=$(readlink -f "$0")
 SCRIPTPATH=$(dirname "$SCRIPT")
 
 helm install \
   --namespace spire-server \
   --values "${SCRIPTPATH}/../../../examples/production/values.yaml" \
-  spire charts/spire
+  spire charts/spire --wait
+
+helm test spire --namespace spire-server
```

Comment:
I understand adding the `--wait`. Was this `helm test` line supposed to be there but omitted before?

Reply:
helm install doesn't do helm test. both are needed and before it was skipping all the tests.

Reply:
just for reference and completeness, most tests use the chart-testing tool to install and test the chart. `ct install` does actually run the tests, but for this test we do a specific helm-intall and therefore we have to run it here as well.